package.json

```diff
@@ -790,52 +790,52 @@
     "webpack-dev": "rimraf dist && webpack --mode development --watch"
   },
   "devDependencies": {
-    "@types/body-parser": "^1.16.8",
-    "@types/express": "^4.11.1",
-    "@types/fs-extra": "^5.0.4",
-    "@types/keytar": "^4.4.0",
-    "@types/mocha": "^2.2.32",
-    "@types/node": "^10.12.3",
-    "@types/terser-webpack-plugin": "^1.2.0",
-    "@types/webpack": "^4.4.24",
-    "copy-webpack-plugin": "^5.0.5",
+    "@types/body-parser": "^1.19.0",
+    "@types/express": "^4.17.7",
+    "@types/fs-extra": "^9.0.1",
+    "@types/keytar": "^4.4.2",
+    "@types/mocha": "^8.0.2",
+    "@types/node": "^14.6.0",
+    "@types/node-fetch": "^2.5.7",
+    "@types/terser-webpack-plugin": "^4.1.0",
+    "@types/webpack": "^4.41.21",
+    "copy-webpack-plugin": "^6.0.3",
     "fail-on-errors-webpack-plugin": "^3.0.0",
-    "mocha": "^5.1.1",
-    "rimraf": "^2.6.2",
+    "keytar": "^6.0.1",
```

Comment:
I think we could remove `keytar` from devDependencies

Reply:
This is the issue that was failing the LInux build.